platform/windows/display_server_windows.cpp

```diff
@@ -4658,7 +4704,7 @@ DisplayServerWindows::DisplayServerWindows(const String &p_rendering_driver, Win
 			SetProcessDpiAwareness_t SetProcessDpiAwareness = (SetProcessDpiAwareness_t)GetProcAddress(Shcore, "SetProcessDpiAwareness");
 
 			if (SetProcessDpiAwareness) {
-				SetProcessDpiAwareness(SHC_PROCESS_SYSTEM_DPI_AWARE);
+				SetProcessDpiAwareness(SHC_PROCESS_PER_MONITOR_DPI_AWARE);
```

Comment:
This parameter is supported only for win 8.1 and higher. As far as I understand, Godot still supports Windows 7. You can copy code with a check from this commit - https://github.com/godotengine/godot/pull/86943/commits/40c7925d2afeb07c982a9b59ff0cbd44a36bfc40